highfidelity/hifi
# Fix transparent textures rendering wrong sometimes

https://highfidelity.fogbugz.com/f/cases/12957/Sandbox-Lights-Show-Texture-Problem

Test plan:
- Go to a domain with lots of things to download.  `dev-mobile-master/-36,30,40` is good.  Turn so you are looking at all the models.  Run [this](https://gist.githubusercontent.com/SamGondelman/819f8b5e77f974a3a0c2114e7510dc38/raw/e8612242f5c5890ef5c5ec16237feba668ed2b8c/ModelTextureTransparency.js).  A lot of teleport pads should appear.  Below each one should be a small yellow "sunbeam".  They should all render as properly transparent.  None of them should appear distorted or opaque (like in the ticket).
- Reload the script multiple times.  Reload content multiple times.  Switch domains and come back.  Clear your KTX Cache (Developer -> Render -> Clear KTX Cache) and restart Interface.  None of the models should ever get stuck with messed up textures, like in the ticket.
- Verify that Model Entity animations still work (flags in dev-welcome)
- Verify that you can still swap textures on a Model Entity.  Select the model in Create mode, copy the "Original Textures" to "Textures" and replace the links with other images from the internet.  The textures on the model should switch.  Clear the "Textures" section and the model should return to normal.
- Verify that you can swap out model URLs and have the model correctly reload (edit a model and give it a new URL, it should update)
